content/_en/help/verify-your-identity/accepted-state-issued-identification.md

```diff
@@ -2,7 +2,7 @@
 layout: help
 title: "Accepted State-Issued Identification "
 category: verify-your-identity
-order: 2
+order: 3
```

Comment:
Do we need to reorder this page and `how-to-verify-your-identity.md` for Spanish and French as well?

Reply:
Yes we do